src/ast_util.py

```diff
@@ -234,21 +234,35 @@ def get_block_stmts(block: Block, force: bool) -> List[Statement]:
     return ret
 
 
-def insert_decl(
-    fn: ca.FuncDef, var: str, type: SimpleType, random: Optional[Random] = None
-) -> None:
+def make_decl(
+    name: str,
+    type: Union[SimpleType, ca.FuncDecl],
+    quals: List[str] = [],
+    align: List[ca.Alignas] = [],
+    storage: List[str] = [],
+    funcspec: List[str] = [],
```

Comment:
It would be preferable to have these be `Optional[...]` and default to None, with `or []` at the place they are used. Reason being that Python evaluates parameter defaults just once instead of once per call, and because lists are mutable that can cause mysterious bugs. Likely wouldn't actually ever come up in practice but it's a good rule to learn.

Reply:
okay